yeastphenome/apps/datasets/templates/datasets/data_table.html

```diff
@@ -1,41 +1,47 @@
 {% if datasets %}
-<style>
-    .tablesorter thead .disabled {
-        display:none;
-    }
-</style>
-<form action="{% url 'datasets:download' %}" method="get">
-    <table id="{{ table_id }}" class="table table-condensed table-hover" width="100%">
-        <thead><tr>
-        <th>Dataset Name</th>
-        <th>Phenotypic Scores</th></tr>
-        </thead>
-        <tbody id="papersTableBody">
-        {% for d in datasets %}<tr>
-            <td class="col-md-5"><a href="/datasets/{{ d.dataset.id }}/">{{ d.dataset.name }}</a></td>
-            <td class="col-md-5">{{ d.value|floatformat:1 }}</td>
-        </tr>{% endfor %}
-        </tbody>
-    </table>
-</form><a><button style="margin-right:5px" class="dataset_download_gene btn btn-info" type="button" class="btn btn-default">Download all</button></a><a href="{% url 'genes:datasets' gene.id %}"><button type="button" class="btn btn-success">View all</button></a>{% endif %}
+    <style>
```

Comment:
For future stuffs, it's hard to see changes if there is change in spacing and content - I can't quickly tell from looking here! If you want to do fixes for just spacing, it would be good to open a PR just for that (to make the reviewing easier!)

Reply:
Yeah, I know. Will try to avoid stylistic changes in parallel to code changes. Here it's a combination of: 
1) auto-formatting (my IDE, for some reason, had very bad default formatting on html files, and I had to run its auto-format function to read the code).
2) change to the 2 buttons to make the both links formatted as buttons (<a class='btn' role='button'></a>), rather than buttons enclosed by links (<a><button></button></a>).

Reply:
Ah, that would do it! I definitely do both ways (for buttons and links) but I just looked it up and you're spot on, the link tag should go inside the button and not vice versa.